tests/test_tokenization_reformer.py

```diff
@@ -328,3 +328,65 @@ def test_torch_encode_plus_sent_to_model(self):
         with torch.no_grad():
             model(**encoded_sequence)
             model(**batch_encoded_sequence)
+
+
+class ReformerByteTokenizationTest(unittest.TestCase):
```

Comment:
I would also at least test saving and loading, the `__call__` method, that it can be sent to a model, padding, truncation, ...